suite-common/suite-types/src/environment.ts

```diff
@@ -1 +1 @@
-export type EnvironmentType = 'web' | 'desktop' | 'mobile' | '';
+export type EnvironmentType = 'web' | 'desktop';
```

Comment:
This type in exactly same shape is also in `@suite-common/analytics`, I think we should remove it in one of places.

Reply:
https://github.com/trezor/trezor-suite/pull/7622/commits/c7f5d2843b15663e8216eaf6606346df93acc59e